tool/clean/clean_host/clean_host.go

```diff
@@ -0,0 +1,93 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build clean
+// +build clean
+
+package main
+
+import (
+	"context"
+	"github.com/aws/amazon-cloudwatch-agent/tool/clean"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/aws/aws-sdk-go/aws"
+	"log"
+	"time"
+)
+
+// Clean integration hosts if they have been open longer than 1 day
+const tagName = "tag:Name"
+var tagValue = []string{
```

Comment:
Tags we have in the internal except soaking

Reply:
If we are using in one place, I would highly recommend not initialize it but put it inside the function that use the variables.

Reply:
Changed